runtime/common/xwalk_content_client.cc

```diff
@@ -1,3 +1,4 @@
+// Copyright (c) 2012 The Chromium Authors. All rights reserved.
```

Comment:
Not sure it is right to add this copyright because we copied code from upstream. cc @baleboy